### PR TITLE
Fix redirects when there is no @wiki.repo.head

### PR DESCRIPTION
**Pull request by dometto**
See https://github.com/gollum/gollum/issues/1530

I'll add a test to gollum checking for an OK response when using en empty repository.

### CODE REVIEW

File: lib/gollum-lib/redirects.rb [hunk 1]
```diff
@@ -33,6 +33,10 @@ def dump
       @wiki.overwrite_file(REDIRECTS_FILE, self.to_yaml, {})
     end
     
+    def get_head_sha
+      @wiki.repo.head.nil? ? nil : @wiki.repo.head.commit.sha
```
Comment (bartkamphorst) [inline lib/gollum-lib/redirects.rb]: Slightly more efficient by not checking explicitly for nil:
`@wiki.repo.head ? @wiki.repo.head.commit.sha : nil`

Comment (dometto) [reply lib/gollum-lib/redirects.rb]: Thanks!
